measures/Measures/MeasFrame.cc

```diff
@@ -415,24 +430,32 @@ Bool MeasFrame::getComet(MVPosition &tdb) const {
 
 void MeasFrame::create() {
   if (!rep) {
-    rep = std::make_shared<FrameRep>();
+    rep = details::MakeCyclic<FrameRep>();
     rep->mymcf = std::make_unique<MCFrame>();
   }
 }
 
 void MeasFrame::fill(const Measure *in) {
   if (in) {
```

Comment:
Please add a comment here why you need to `Freeze()` and `UnFreeze()` here. As you explained in the description of this PR, the dependency cycle isn't trivial.

Reply:
Agree; though I don't know 100% sure why some things need the frozen counter (they're coming from the 'lock' statements that were there before #1426), but I've added the possible cycles I identified.